share/completions/pfctl.fish

```diff
@@ -0,0 +1,32 @@
+complete -c pfctl -s s --description 'Show a filter parameter by modifier' -xa \
+	'queue\t"Show the currently loaded queue definitions." \
+	rules\t"Show the currently loaded filter rules." \
+	Anchors\t"Show the currently loaded anchors directly attached to the main ruleset." \
```

Comment:
Descriptions should be as short as possible, so we can ideally fit more columns - `Show anchors attached to main ruleset`?

Also, I prefer no trailing ".".